apis/v1beta1/config_types.go

```diff
@@ -62,8 +62,18 @@ type Roles struct {
 	Custom     []NodeRole `json:"custom,omitempty"`
 }
 
+type VeleroBuiltin struct {
```

Comment:
Can we have some comments on these new structs explaining why they exist ? Maybe showing up an use case.

Reply:
sure